tests/integration/components/popover/event-bubbling-test.js

```diff
@@ -0,0 +1,70 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import {
+  assertTooltipNotVisible,
+  assertTooltipRendered,
+  triggerTooltipTargetEvent,
+} from '../../../helpers/ember-tooltips';
```

Comment:
should we use the syntax we recommend in the README?

`import { ... } from <app-name/helpers/ember-tooltips>`